app/models/restaurant.rb

```diff
@@ -8,4 +8,20 @@ class Restaurant < ActiveRecord::Base
   validates :city, presence: true
   validates :state, presence: true
   validates :zip_code, presence: true
+
+  def average_rating
+    self.reviews.sum(:rating)/self.reviews.count
```

Comment:
`self` may be redundant here - check if there's the same functionality when removing self (I may be wrong here)